model/testing/src/icon4py/model/testing/grid_utils.py

```diff
@@ -10,9 +10,10 @@
 
 import gt4py.next as gtx
 import gt4py.next.backend as gtx_backend
+from gt4py._core import locking
```

Comment:
I am always wondering what should actually be used from gt4py...

Reply:
this not!